Data/Classes/ProfessionGear.lua

```diff
@@ -34,13 +34,20 @@ function CraftSim.ProfessionGear:SetItem(itemLink)
 	end
 
 	self.item = Item:CreateFromItemLink(itemLink)
+	itemLink = self.item:GetItemLink()

```

Comment:
Is this necessary here? shouldnt the itemLink already be in the itemLink parameter of the function?

Reply:
Again, that's just a test i forgot to delete, sorry ^^
Trying to test if i load a second time the item link, maybe i have a full item link. But no